go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/ossf/malicious-packages
 
-go 1.21
+go 1.21.11
+
+toolchain go1.22.4
```

Comment:
Are the toolchain directive and go patch version number changes intentional?

Reply:
Not really. They're from updating osv-scanner.

For what it's worth they don't make a whole lot of difference to this project, so I am fine with them.